docs/running_clusterfuzzlite.md

```diff
@@ -116,75 +76,71 @@ serves two important purposes:
 Over time, redundant testcases will get introduced into your fuzzer's corpuses
 during [batch fuzzing].
 
-Corpus pruning minimizes the corpuses by removing corpus files (testcases) that
+Corpus pruning is a helper function that minimizes the corpuses by removing corpus files (testcases) that
 do not increase the fuzzer's code coverage.
 
-Running corpus pruning once a day will prevent buildup of these redundant
-testcases and keep fuzzing efficient.
+If you are using [batch fuzzing], you should run corpus pruning once a day to prevent buildup of these redundant
+testcases and keep fuzzing efficient. Corpus pruning should not be used in conjunction with modes other than [batch fuzzing].
```

Comment:
This line about conjuction doesn't really make sense.
It's mandatory if you have batch fuzzing, otherwise it's pointless.

Reply:
How about we reinstate something close to the old version:

`Corpus pruning should be
considered mandatory when you are using [batch fuzzing] but otherwise should not be
used.`

Reply:
+1